src/ui/Header.tsx

```diff
@@ -1,18 +1,18 @@
 import styled from '@emotion/styled';
 import { Link } from 'react-router-dom';
 import { FC, useState } from 'react';
+import Flex from './Flex';
 
 const StyledHeader = styled.header`
   width: 100%;
-  height: 50px;
-  background-color: var(--background);
+  height: 30px;
+  background-color: var(--darkest-wine);
   font-weight: 600;
```

Comment:
typography styles should be abstracted to semantic components in a Typography.css or App.css file. i'll let this slide for now, but we should update it in the future.

the designers should specify all the different typographies, hopefully only a handful, then we should only define them once